### DISCUSSION TITLE
part-ii/2020-6/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#123] [Question comments]
Part II 2020 Algebraic Topology: Paper 3, Section II, 20F 

https://questions.tripos.org/part-ii/2020-6/

**Comment by artie2000** [comment]
(b): $H_0=\mathbb{Z}$, $H_1=\mathbb{Z}$ (others vanish).

<details>
<summary>(c): Set</summary>

$h(v_3)=v_{13}$, $h(v_{23})=-v_{123}$, and $h=0$ on the orthogonal complement.

</details>
I found it easiest to figure out the signs directly from $dh+hd=f-g$.

**Reply by ArthurConmy** [reply]
This is with $ j(v_3) = v_1$, right? 

($j(v_3)=v_2$ is also possible I think)

**Reply by artie2000** [reply]
Ah, yes!
The construction should be pretty much the same in the other case.
